src/components/article/articleComponent/tagComponent.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const TagComponent = (props) => {
+  const { tag } = props;
+  return (
+    <Link className="mt-1" to="#">{tag}</Link>
```

Comment:
Do you intend to have the individual tags clickable at some point??

Reply:
Yes @wasibani-roy, Later we will want to click on a link that displays all articles with a specific tag